docs/source/guide.rst

```diff
@@ -138,11 +138,10 @@ The hook will retrieve a balance of an ERC20 token of the provided address.
     address: string | Falsy
   ) {
     const [tokenBalance] =
-      useContractCall(
+      useCall(
         address &&
           tokenAddress && {
-            abi: ERC20Interface, // ABI interface of the called contract
-            address: tokenAddress, // On-chain address of the deployed contract
+            contract: contract: new Contract(tokenAddress, ERC20Interface), // instance of called contract
```

Comment:
Duplicated: `contract:`

Reply:
Feature to consider in future PRs:
contract: createContract(tokenAddress, ERC20Interface)